.github/workflows/build.yaml

```
@@ -0,0 +1,222 @@
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        service: [ "${{ github.event.inputs.service }}" ]
+
+    outputs:
+      docker_image: ${{ steps.save_docker_image.outputs.docker_image }}
+      db_image:     ${{ steps.save_db_image.outputs.db_image }}
```

YAMLlint (1.37.1)

[error] too many spaces after colon

(colons)